.github/CODEOWNERS

```diff
@@ -0,0 +1,105 @@
+/clib/            @ppedrot
+
+/doc/             @silene @maximedenes
+/man/             @silene @maximedenes
+
+/engine/          @ppedrot @aspiwack
+
+# We need a secondary maintainer for grammar
+/grammar/         @ppedrot
+
+/ide/             @ppedrot @gares
+
+# We need a maintainer for interp
+#/interp/
+
+/intf/            @letouzey @ppedrot
+/kernel/          @maximedenes @barras
+/lib/             @ejgallego @ppedrot
+/library/         @silene @tassi
+
+/parsing/         @herbelin @mattam82
+
+# We need secondary maintainers for many of these
+/plugins/btauto/  @ppedrot
+
+# I don't know Pierre Corbineau's GitHub nickname
+/plugins/cc/            @herbelin
+/plugins/derive/        @aspiwack @ppedrot
+/plugins/extraction/    @letouzey @maximedenes
+# I don't know Pierre Corbineau's GitHub nickname
+/plugins/firstorder/    @herbelin
+/plugins/fourier/       @herbelin
+# I don't know Julien Forest's GitHub nickname
+/plugins/funind/        @Matafou
+/plugins/ltac/          @ppedrot
+/plugins/micromega/     @fajb
+/plugins/nsatz/         @thery
+/plugins/omega/         @letouzey
+/plugins/romega/        @letouzey
+/plugins/setoid_ring/   @amahboubi
+/plugins/ssrmatching/   @gares @maximedenes
+/plugins/ssr/           @gares @maximedenes
+/plugins/syntax/        @ppedrot
+
+
+# We need a maintainer for quote
+#/plugins/quote/
+
+# Should be Pierre Corbineau and someone else
+#/plugins/rtauto
+
+/pretyping/       @mattam82 @gares
+/printing/        @herbelin @mattam82
+
+# We need a secondary maintainer for proofs
+/proofs/          @ppedrot
+
+/stm/             @gares @ejgallego
+
+/tactics/         @ppedrot @mattam82
+
+# We need a secondary maintainer for many of these
+/theories/Arith/          @letouzey
+/theories/Bool/           @letouzey
+/theories/Classes/        @mattam82
+/theories/FSets/          @letouzey
+/theories/Init/           @letouzey @ppedrot
+/theories/Lists/          @letouzey @ppedrot
+/theories/Logic/          @herbelin @ppedrot
+/theories/MSets/          @letouzey
+/theories/NArith/         @letouzey
+/theories/Numbers/        @letouzey
+/theories/PArith/         @letouzey
+/theories/Program/        @mattam82
+/theories/QArith/         @letouzey
+/theories/Reals/          @silene @ppedrot
+/theories/Relations/      @mattam82 @ppedrot
+/theories/Setoids/        @mattam82 @ppedrot
+/theories/Sets/           @letouzey
+/theories/Sorting/        @letouzey
+/theories/Strings/        @letouzey @herbelin
+/theories/Structures/     @letouzey
+/theories/Unicode/        @letouzey @herbelin
+/theories/Wellfounded/    @letouzey @mattam82
+/theories/ZArith/         @letouzey
+
+# We need a maintainer for these
+#/theories/Compat/
+#/theories/Vectors/
+
+/tools/coqdoc/          @silene @mattam82
+/tools/coq_makefile*    @gares
+/tools/CoqMakefile*     @gares
+/tools/coqdep*          @ppedrot @maximedenes
+/tools/coq_tex*         @silene
+/tools/coqwc*           @silene
+
+/toplevel/        @ejgallego @gares
```

Comment:
`toplevel` contains a lot of boring code everybody understands, + a few tricky bits about initialization that I may know better.  So I'm fine having my name here.

`vernac` I really don't know why my name is in there:
```
vernac/assumptions.mli         vernac/declareDef.mli   vernac/proof_using.mli
vernac/auto_ind_decl.mli       vernac/explainErr.mli   vernac/record.mli
vernac/classes.mli             vernac/himsg.mli        vernac/search.mli
vernac/class.mli               vernac/indschemes.mli   vernac/topfmt.mli
vernac/comAssumption.mli       vernac/lemmas.mli       vernac/vernacentries.mli
vernac/comDefinition.mli       vernac/locality.mli     vernac/vernacinterp.mli
vernac/comFixpoint.mli         vernac/metasyntax.mli   vernac/vernacprop.mli
vernac/comInductive.mli        vernac/mltop.mli        vernac/vernacstate.mli
vernac/comProgramFixpoint.mli  vernac/obligations.mli
```
The only (trivial) file in there I wrote is `proof_using`, and the only code I've touched is the dispatch of the interpreter. All the rest is the implementation of vernacular commands. I'm not comfortable with my name being here. For sure it lacks @mattam82 for program/obligations. It should also contain the code of Arguments, that @maximedenes completely rewrote, doesn't it?